etest/docker/container.py

```diff
@@ -6,15 +6,21 @@
 
 from typing import List
 
+from docker.models.containers import Container
+
 from etest.docker import common
 
 CONTAINERS: List = []
 CREATE = True
 
 
-def commit(*args, **kwargs):
+def commit(container: Container, tag: str, repo_: str, *args, **kwargs):
```

Comment:
The interface should have the non-mangled name for a better consumer API.  Let's change repo_ to repository here.

Reply:
Sure, I added that to differentiate it from the local scoped repo